.gitignore

```diff
@@ -67,3 +67,7 @@ forge-artifacts
 # Compiler files
 cache
 out
+
+# Presigning and deployment directories
+presigned-createx-deployment-transaction
+broadcasted-createx-deployment-transaction
```

Comment:
Why are these gitignored, don't we want anyone to be able to deploy the presigned transaction?

Reply:
That's just temporary until we froze the code and generated the final transaction. I don't want to pollute the commit history with presigned transactions that are actually not valid. Also, it makes it safe for local testing without accidentally pushing a presigned transaction.